base/array.jl

```diff
@@ -1473,6 +1473,9 @@ julia> a[1:6]
 ```
 """
 function resize!(a::Vector, nl::Integer)
+    resize!(a, Int(nl)::Int)
+end
+function resize!(a::Vector, nl::Int)
```

Comment:
Do we need this extra method, or can we convert the size to `Int` within the generic method that accepts an `Integer` size?